pnpm-lock.yaml

```diff
@@ -11344,6 +11336,9 @@ packages:
   picocolors@1.0.0:
     resolution: {integrity: sha512-1fygroTLlHu66zi26VoTDv8yRgm0Fccecssto+MhsZ0D/DGW2sm8E8AjW7NU5VVTRt5GxbeZ5qBuJr+HyLYkjQ==}
 
+  picocolors@1.0.1:
+    resolution: {integrity: sha512-anP1Z8qwhkbmu7MFP5iTt+wQKXgwzf7zTyGlcdzabySa9vd0Xt392U0rVmz9poOaBj0uHJKyyo9/upk0HrEQew==}
+
```

Comment:
Hmm, failed deduplication? Is there somewhere (upstream?) a missing caret range?

Reply:
Whoops, it's create-next-app. Didn't realise we were specifying exact versions there.